src/interactive/AccordionPanel.jsx

```diff
@@ -30,49 +25,33 @@ class AccordionPanel extends React.Component {
 	 * @returns {Number} panel height
 	 */
 	getHeight(isOpen) {
+		if (!this.contentEl) {
+			return 'auto';
```

Comment:
@mperrotti is this a kosher default? or is `0px` better.

Reply:
Since we animate height - `0px` is better. You can't animate from `auto`